src/ray/raylet/local_object_manager.cc

```diff
@@ -261,51 +261,26 @@ void LocalObjectManager::SpillObjectsInternal(
                 auto it = objects_pending_spill_.find(object_id);
                 RAY_CHECK(it != objects_pending_spill_.end());
                 pinned_objects_size_ += it->second.first->GetSize();
+                num_bytes_pending_spill_ -= it->second.first->GetSize();
```

Comment:
This was a bug. We have one more bug in stats update path. We can talk about it more @scv119  later

Reply:
yeah, trying to understand the behavior change of this PR. we can chat on Monday.